.env

```diff
@@ -65,6 +65,11 @@ CONCEPT_COMPLEX_UUID=
 OPENMRS_IDENTIFIER_TYPE_UUD=
 DB_EVENT_DESTINATIONS=direct:odoo-event-listener
 CREATE_CUSTOMER_IF_NOT_EXIST=false
+ODOO_CUSTOMER_EXTERNAL_ID=x_external_identifier
```

Comment:
Is this not a field? I'm surprised that the other vars are prefixed `_FIELD` and not this one. Is it different?
Additionally, External ID means something quite specific in Odoo (a mapping table where references between entities and values are assigned). Are you referring to this?
If not, and you're simply referring to a field in Odoo (maybe on on the res_partner table), then it would be best to name this to for instance: `PATIENT_IDENTIFIER_FIELD`.

What do you think?

Reply:
Thanks for the review @rbuisson. I agree to all the above suggestions. Could I apply the above modifications through separate PRs? This is because other repos are affected by the suggestions. I'd like to let the introduced features tested already by @nuhuhmutebi since some follow-up commits might be required in the end.

Reply:
Let me merge then.